pyms/flask/configreload/configreload.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals, print_function, absolute_import, division
+from flask import Blueprint
+
+configreload_blueprint = Blueprint('configreload', __name__, static_url_path='/static')
+
+
+@configreload_blueprint.route('/reload_config', methods=['GET'])
```

Comment:
I would suggest to change the method to post. The de facto reloader on k8s expects post: https://github.com/jimmidyson/configmap-reload. @lcd1232

Reply:
@alexppg done

Reply:
One more thing:
Use hyphens (`-`) instead of underscores (`_`)

More info: https://restfulapi.net/resource-naming/

Good PR @lcd1232! ^^

Reply:
Done